extmod/modujson.c

```diff
@@ -224,6 +224,12 @@ STATIC mp_obj_t mod_ujson_load(mp_obj_t stream_obj) {
                 break;
             case '{':
                 next = mp_obj_new_dict(0);
+                #if MICROPY_PY_UJSON_ORDERED_DICT
```

Comment:
Is this a new option? Is it set anywhere?

Reply:
It is a new option, and no it's not set by default anywhere. The expectation is it'll be set in make option or board profile if someone wants it... or if it's a desieable enough feature it could be made the default

Reply:
Hmm, doesn't that make it almost impossible to discover? Someone has to go look at the code to see "hey, look at that! I can enable MICROPY_PY_UJSON_ORDERED_DICT!". Just wondering...

Reply:
Yeah it's certainly not ideal... At the very least I should enable it on the coverage build and add a unit test - this initial PR was thrown up very quickly in the middle of a project rebase/test/release cycle on my end and I didn't want it to be forgotten.

Reply:
Just put in in mpconfig.h, like all other options.